moped-editor/src/views/projects/NewProjectView.js

```diff
@@ -172,13 +173,15 @@ const NewProjectView = () => {
       $role_name: String! = ""
       $first_name: String! = ""
       $last_name: String! = ""
+      $notes: String! = ""
```

Comment:
I did not know you could have defaults in a hasura graphql query 🙌  this is excellent!